api/v1alpha2/awscluster_conversion.go

```diff
@@ -46,6 +74,11 @@ func (dst *AWSClusterList) ConvertFrom(srcRaw conversion.Hub) error { // nolint
 	return Convert_v1alpha3_AWSClusterList_To_v1alpha2_AWSClusterList(src, dst, nil)
 }
 
+// Convert_v1alpha2_AWSClusterStatus_To_v1alpha3_AWSClusterStatus is an autogenerated conversion function.
```

Comment:
I think this comment needs to be updated to indicate that it's not autogenerated

Reply:
good catch